modules/evm/src/lib.rs

```diff
@@ -882,6 +885,16 @@ impl<T: Config> EVMTrait for Module<T> {
 			}
 		})
 	}
+
+	/// Get the real origin account and charge storage rent from the origin.
+	fn get_origin() -> T::AccountId {
+		ExtrinsicOrigin::<T>::get().expect("There must be a value through `SignedExtension::pre_dispatch`")
```

Comment:
this can be called in `on_intialize` / `on_finalize` so there could be no origin. so should make it return `Option<T::AccountId>` instead

And in module-currencies, use unwrap_or_default when get origin